packages/design-system/src/components/Table/Table.tsx

```diff
@@ -107,7 +111,7 @@ export class Table extends React.Component<
     this.state = {
       scrollActive: false,
     };
-    this.captionID = uniqueId('caption-');
+    this.captionId = props.id ?? uniqueId('table-caption--');
```

Comment:
I corrected the "ID" part of this const to be consistent with the rest of our usage ("Id")